drivers/CMakeLists.txt

```diff
@@ -41,6 +41,7 @@ add_subdirectory_ifdef(CONFIG_LORA lora)
 add_subdirectory_ifdef(CONFIG_PECI peci)
 add_subdirectory_ifdef(CONFIG_REGULATOR regulator)
 add_subdirectory_ifdef(CONFIG_MEMC memc)
+add_subdirectory_ifdef(CONFIG_MISC misc)
```

Comment:
CONFIG_MISC is really a wildcard here and can be anything. We should find an existing category or create a new one. Maybe ipm? or ipc? or even shared_memory, anything, but not MISC please :-)

Reply:
I was out of inspiration, I admit. virtualization? guest? Maybe the first. I wouldn't be surprised to see more virtualization/emulation stuff landing in zephyr at some point.

Reply:
made it to virtualization